core/src/main/java/io/grpc/LoadBalancer.java

```diff
@@ -450,9 +419,8 @@ public boolean equals(Object other) {
      * @throws IllegalArgumentException if {@code subchannel} was not returned from {@link
      *     #createSubchannel}
      */
-    public void updateSubchannelAddresses(Subchannel subchannel, EquivalentAddressGroup addrs) {
-      throw new UnsupportedOperationException();
-    }
+    public abstract void updateSubchannelAddresses(
+        Subchannel subchannel, EquivalentAddressGroup addrs);
```

Comment:
This change is unrelated to the deletion of the deprecated methods, and it doesn't seem necessary.

Reply:
Sorry, abstracted the wrong method